packages/beeq/src/components/dialog/bq-dialog.tsx

```diff
@@ -237,8 +246,12 @@ export class BqDialog {
   // ===================================
 
   render() {
+    const style = {
+      ...(this.border && { '--bq-dialog--border-radius': `var(--bq-radius--${this.border})` }),
+    };
```

Comment:
Please leave an empty line between the const declaration and the return statement.